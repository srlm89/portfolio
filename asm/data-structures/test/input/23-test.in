ERASE actual/23-test.out
INSERT 18 1.475254 Bugs.A.Rainforest.Adventure.3D.HD.DVD.REMUX.1080p.VC-1-SEBUL
PRINT_IN_ORDER actual/23-test.out
INSERT 7 8.643580 Pinocchio [1940] [720p]
INSERT 3 5.471752 Lord Of The Ring  - The Two Tower   720p
REMOVE_ID 3
PRINT_IN_ORDER actual/23-test.out
INSERT 9 1.413862 Shoot.Em.Up.720p.Bluray.x264-SEPTiC
INSERT 20 7.399000 James Bond 12 - For Your Eyes Only [1981]
INSERT 0 4.264240 The Matrix [1999] [1080p]
REMOVE_ID 9
PRINT_IN_ORDER actual/23-test.out
INSERT 17 8.305952 Danny the Dog   1080p
INSERT 30 7.460994 Juno 2007 720p
INSERT 13 5.836945 The Chronicle of Riddick 720p
INSERT 19 2.110111 Toy.Story.2.HD.720p.Hebrew.Dubbed.Also.English.x264-ALiS_20
REMOVE_ID 18
REMOVE_ID 0
PRINT_IN_ORDER actual/23-test.out
INSERT 25 4.409041 A.Beautiful.Mind.2001.720p.HDTV.x264-S26
INSERT 2 3.238601 Chocolate 720p
INSERT 23 1.543418 Crossing Over 720P
INSERT 32 1.037735 Fight.Club.1999.720p.BluRay.DTS-ES.x264-DON
INSERT 6 7.438647 Pursuit_Of_Happyness[2006]   720p
REMOVE_ID 30
REMOVE_ID 7
PRINT_IN_ORDER actual/23-test.out
INSERT 16 0.180553 Dante's Peak   1080p
INSERT 39 0.131913 Cellular.2004.HDTV.720p.AC3.x264-BTT
INSERT 21 7.653940 Moulin Rouge   720p
INSERT 10 6.916379 X-Men 1    720p
INSERT 29 8.620469 James Bond 15 - The Living Daylights [1987] [720p]
INSERT 8 9.838966 Basic Instinct 720p
REMOVE_ID 29
REMOVE_ID 10
REMOVE_ID 39
PRINT_IN_ORDER actual/23-test.out
INSERT 26 0.228980 Shark Tale [2004] [1080p]
INSERT 22 2.580645 Fantastic Four   1080p
INSERT 12 0.017533 Diary.Of.A.Sex.Addict.2008.720p.BluRay.x264-FLHD
INSERT 41 3.074539 Die Hard 4   1080p
INSERT 43 5.028300 Beauty And The Beast    720p
INSERT 24 7.462954 Bugs.A.Rainforest.Adventure.3D.HD.DVD.REMUX.1080p.VC-1-SEBUL
INSERT 33 4.690767 The Matrix Revolution   1080p
REMOVE_ID 24
REMOVE_ID 25
REMOVE_ID 2
PRINT_IN_ORDER actual/23-test.out
INSERT 51 6.179252 The.Shawshank.Redemption.1994.1080p.BluRay.x264-CiNEFiLE
INSERT 31 9.512360 Invisible.Target.720p.Bluray.x264-SEPTiC
INSERT 27 6.214685 Se7en [720p]
INSERT 37 1.525550 Stargate.Continuum.2008.1080p.Bluray.DTS.x264-eVIAN
INSERT 36 5.191903 Die Hard 3   720p
INSERT 54 4.408471 Counterfeiters.LIMITED.720p.Bluray.x264-SEPTiC
INSERT 53 5.459075 Batman The Dark Knight   720p
INSERT 15 1.174994 A Bug's Life [720p]
REMOVE_ID 54
REMOVE_ID 32
REMOVE_ID 13
REMOVE_ID 26
PRINT_IN_ORDER actual/23-test.out
INSERT 1 0.320515 Madagascar 2 Escape 2 Africa [2008] [720p]
INSERT 58 4.539233 Die Hard 3   720p
INSERT 40 9.409960 Big Buck Bunny [1080p]
INSERT 45 1.106910 X-Men 2   1080p
INSERT 57 7.480154 Ice Age 2 The Meltdown [2006] [720p]
INSERT 4 1.435899 Tomb Raider Collection (1-2)    720p
INSERT 63 7.115159 Dragonball Evolution [2009] [720p]
INSERT 38 9.648286 How To Lose Friends & Alienate People   1080p
INSERT 14 0.375198 PIXAR Movie
REMOVE_ID 14
REMOVE_ID 15
REMOVE_ID 27
REMOVE_ID 6
PRINT_IN_ORDER actual/23-test.out
