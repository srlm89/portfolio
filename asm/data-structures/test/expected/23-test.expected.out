[ 18 1.475254 Bugs.A.Rainforest.Adventure.3D.HD.DVD.REMUX.1080p.VC-1-SEBUL ]
[ 7 8.643580 Pinocchio [1940] [720p] ][ 18 1.475254 Bugs.A.Rainforest.Adventure.3D.HD.DVD.REMUX.1080p.VC-1-SEBUL ]
[ 0 4.264240 The Matrix [1999] [1080p] ][ 7 8.643580 Pinocchio [1940] [720p] ][ 18 1.475254 Bugs.A.Rainforest.Adventure.3D.HD.DVD.REMUX.1080p.VC-1-SEBUL ][ 20 7.399000 James Bond 12 - For Your Eyes Only [1981] ]
[ 7 8.643580 Pinocchio [1940] [720p] ][ 13 5.836945 The Chronicle of Riddick 720p ][ 17 8.305952 Danny the Dog   1080p ][ 19 2.110111 Toy.Story.2.HD.720p.Hebrew.Dubbed.Also.English.x264-ALiS_20 ][ 20 7.399000 James Bond 12 - For Your Eyes Only [1981] ][ 30 7.460994 Juno 2007 720p ]
[ 2 3.238601 Chocolate 720p ][ 6 7.438647 Pursuit_Of_Happyness[2006]   720p ][ 13 5.836945 The Chronicle of Riddick 720p ][ 17 8.305952 Danny the Dog   1080p ][ 19 2.110111 Toy.Story.2.HD.720p.Hebrew.Dubbed.Also.English.x264-ALiS_20 ][ 20 7.399000 James Bond 12 - For Your Eyes Only [1981] ][ 23 1.543418 Crossing Over 720P ][ 25 4.409041 A.Beautiful.Mind.2001.720p.HDTV.x264-S26 ][ 32 1.037735 Fight.Club.1999.720p.BluRay.DTS-ES.x264-DON ]
[ 2 3.238601 Chocolate 720p ][ 6 7.438647 Pursuit_Of_Happyness[2006]   720p ][ 8 9.838966 Basic Instinct 720p ][ 13 5.836945 The Chronicle of Riddick 720p ][ 16 0.180553 Dante's Peak   1080p ][ 17 8.305952 Danny the Dog   1080p ][ 19 2.110111 Toy.Story.2.HD.720p.Hebrew.Dubbed.Also.English.x264-ALiS_20 ][ 20 7.399000 James Bond 12 - For Your Eyes Only [1981] ][ 21 7.653940 Moulin Rouge   720p ][ 23 1.543418 Crossing Over 720P ][ 25 4.409041 A.Beautiful.Mind.2001.720p.HDTV.x264-S26 ][ 32 1.037735 Fight.Club.1999.720p.BluRay.DTS-ES.x264-DON ]
[ 6 7.438647 Pursuit_Of_Happyness[2006]   720p ][ 8 9.838966 Basic Instinct 720p ][ 12 0.017533 Diary.Of.A.Sex.Addict.2008.720p.BluRay.x264-FLHD ][ 13 5.836945 The Chronicle of Riddick 720p ][ 16 0.180553 Dante's Peak   1080p ][ 17 8.305952 Danny the Dog   1080p ][ 19 2.110111 Toy.Story.2.HD.720p.Hebrew.Dubbed.Also.English.x264-ALiS_20 ][ 20 7.399000 James Bond 12 - For Your Eyes Only [1981] ][ 21 7.653940 Moulin Rouge   720p ][ 22 2.580645 Fantastic Four   1080p ][ 23 1.543418 Crossing Over 720P ][ 26 0.228980 Shark Tale [2004] [1080p] ][ 32 1.037735 Fight.Club.1999.720p.BluRay.DTS-ES.x264-DON ][ 33 4.690767 The Matrix Revolution   1080p ][ 41 3.074539 Die Hard 4   1080p ][ 43 5.028300 Beauty And The Beast    720p ]
[ 6 7.438647 Pursuit_Of_Happyness[2006]   720p ][ 8 9.838966 Basic Instinct 720p ][ 12 0.017533 Diary.Of.A.Sex.Addict.2008.720p.BluRay.x264-FLHD ][ 15 1.174994 A Bug's Life [720p] ][ 16 0.180553 Dante's Peak   1080p ][ 17 8.305952 Danny the Dog   1080p ][ 19 2.110111 Toy.Story.2.HD.720p.Hebrew.Dubbed.Also.English.x264-ALiS_20 ][ 20 7.399000 James Bond 12 - For Your Eyes Only [1981] ][ 21 7.653940 Moulin Rouge   720p ][ 22 2.580645 Fantastic Four   1080p ][ 23 1.543418 Crossing Over 720P ][ 27 6.214685 Se7en [720p] ][ 31 9.512360 Invisible.Target.720p.Bluray.x264-SEPTiC ][ 33 4.690767 The Matrix Revolution   1080p ][ 36 5.191903 Die Hard 3   720p ][ 37 1.525550 Stargate.Continuum.2008.1080p.Bluray.DTS.x264-eVIAN ][ 41 3.074539 Die Hard 4   1080p ][ 43 5.028300 Beauty And The Beast    720p ][ 51 6.179252 The.Shawshank.Redemption.1994.1080p.BluRay.x264-CiNEFiLE ][ 53 5.459075 Batman The Dark Knight   720p ]
[ 1 0.320515 Madagascar 2 Escape 2 Africa [2008] [720p] ][ 4 1.435899 Tomb Raider Collection (1-2)    720p ][ 8 9.838966 Basic Instinct 720p ][ 12 0.017533 Diary.Of.A.Sex.Addict.2008.720p.BluRay.x264-FLHD ][ 16 0.180553 Dante's Peak   1080p ][ 17 8.305952 Danny the Dog   1080p ][ 19 2.110111 Toy.Story.2.HD.720p.Hebrew.Dubbed.Also.English.x264-ALiS_20 ][ 20 7.399000 James Bond 12 - For Your Eyes Only [1981] ][ 21 7.653940 Moulin Rouge   720p ][ 22 2.580645 Fantastic Four   1080p ][ 23 1.543418 Crossing Over 720P ][ 31 9.512360 Invisible.Target.720p.Bluray.x264-SEPTiC ][ 33 4.690767 The Matrix Revolution   1080p ][ 36 5.191903 Die Hard 3   720p ][ 37 1.525550 Stargate.Continuum.2008.1080p.Bluray.DTS.x264-eVIAN ][ 38 9.648286 How To Lose Friends & Alienate People   1080p ][ 40 9.409960 Big Buck Bunny [1080p] ][ 41 3.074539 Die Hard 4   1080p ][ 43 5.028300 Beauty And The Beast    720p ][ 45 1.106910 X-Men 2   1080p ][ 51 6.179252 The.Shawshank.Redemption.1994.1080p.BluRay.x264-CiNEFiLE ][ 53 5.459075 Batman The Dark Knight   720p ][ 57 7.480154 Ice Age 2 The Meltdown [2006] [720p] ][ 58 4.539233 Die Hard 3   720p ][ 63 7.115159 Dragonball Evolution [2009] [720p] ]
